### PR TITLE
Issue/48 Stop creation of empty testing entity

### LINKED ISSUE #48: Stop creation of empty watcher when closing watcher creation window
As a <kbd>user</kbd> I *don't want an empty watcher to be created* when i cancel the creation, *in order not to have to remove it afterwards*.

### Acceptance criteria

- [ ] Upon pressing the red x to close the window for creating a watcher, no watchers are created.

### Effort Evaluation

Points: 4

### PR DESCRIPTION
**Pull request by SebastianWagner2**
Closes: #48
Contributes to: #65 

Heavy Refactoring in GUI, creation of empty testing entity fixed

### CODE REVIEW

**Comment by SebastianWagner2** [comment]
The signature of many methods has already been and will still be changed because of refactoring and raising the level of abstraction. As a result of this, tests using old methods/method signatures are now failing. I suggest fixing and writing tests after the architectural and inter-method refactoring is through.

**Comment by kolioOtSofia** [comment]
> The signature of many methods has already been and will still be changed because of refactoring and raising the level of abstraction. As a result of this, tests using old methods/method signatures are now failing. I suggest fixing and writing tests after the architectural and inter-method refactoring is through.

I think it was gonna be nice to have the tests working, so that we can test whether we broke the old functionalities with our code, but I assume that we can also test this by clicking all the stuff. However, my suggestion would be to merge this PR last, so that we can have working tests till the end of the sprint.

**Review by kolioOtSofia — APPROVED** [review]

File: packages/AutoTDD-Core.package/ATDDRemoteTestingEntity.class/instance/token.st [hunk 1]
```diff
@@ -0,0 +1,4 @@
+as yet unclassified
```
Comment (Toalaah) [inline packages/AutoTDD-Core.package/ATDDRemoteTestingEntity.class/instance/token.st]: Is this supposed to be unclassified?

File: packages/AutoTDD-Core.package/ATDDRemoteTestingEntity.class/instance/token..st [hunk 2]
```diff
@@ -0,0 +1,4 @@
+as yet unclassified
```
Comment (Toalaah) [inline packages/AutoTDD-Core.package/ATDDRemoteTestingEntity.class/instance/token..st]: Is this supposed to be unclassified?

File: packages/AutoTDD-Core.package/ATDDWatcher.class/class/symbol.st [hunk 3]
```diff
@@ -0,0 +1,4 @@
+as yet unclassified
```
Comment (Toalaah) [inline packages/AutoTDD-Core.package/ATDDWatcher.class/class/symbol.st]: Is this supposed to be unclassified?

File: packages/AutoTDD-GUI.package/ATDDBrowser.class/class/travisStatusFunctionalityDictionary.st [hunk 4]
```diff
@@ -3,12 +3,12 @@ travisStatusFunctionalityDictionary
...
+			'Pause' -> #pauseTravisStatuses.
 			'Resume' -> #resumeSelectedTravisStatuses.
 			'Run' -> #resumeSelectedTravisStatuses.
-      		'Edit' -> #editTravisStatus.
-			'Info' -> #openInfoForSelectedTravisStatuses.
-			'Results' -> #openResultsForSelectedTravisStatuses.
+      		'Edit' -> #editTravisStatuses.
+			'Info' -> #infoTravisStatuses.
+			'Results' -> #resultsTravisStatuses.
 			'Remove' -> #removeSelectedTravisStatuses.
```
Comment (Toalaah) [inline packages/AutoTDD-GUI.package/ATDDBrowser.class/class/travisStatusFunctionalityDictionary.st]: We should probably be consistent with periods on the last line of ordered dictionaries. For instance, this function does have a period on the last line, however the method `testingEntityResultsDictionary` from the same class does not. Perhaps there is something I'm not seeing?

File: packages/AutoTDD-GUI.package/ATDDBrowser.class/class/watcherFunctionalityDictionary.st [hunk 5]
```diff
@@ -3,12 +3,12 @@ watcherFunctionalityDictionary
...
+			'Pause' -> #pauseWatchers.
 			'Resume' -> #resumeSelectedWatchers.
 			'Run' -> #resumeSelectedWatchers.
-      		'Edit' -> #editWatcher.
-			'Info' -> #openInfoForSelectedWatchers.
-			'Results' -> #openResultsForSelectedWatchers.
+      		'Edit' -> #editWatchers.
+			'Info' -> #infoWatchers.
+			'Results' -> #resultsWatchers.
 			'Remove' -> #removeSelectedWatchers.
```
Comment (Toalaah) [inline packages/AutoTDD-GUI.package/ATDDBrowser.class/class/watcherFunctionalityDictionary.st]: We should probably be consistent with periods on the last line of ordered dictionaries. For instance, this function does have a period on the last line, however the method `testingEntityResultsDictionary` from the same class does not. Perhaps there is something I'm not seeing?

**Review by Toalaah — APPROVED** [review]

**Pull request merged**